src/libcore/num/f32.rs

```diff
@@ -553,7 +552,6 @@ macro_rules! f32_core_methods { () => {
 
 #[lang = "f32"]
 #[cfg(not(test))]
-#[cfg(not(stage0))]
 impl f32 {
     f32_core_methods!();
```

Comment:
I think the idea is that the macro above would be inlined here but we can always do that in a follow-up